src/azure-cli/azure/cli/command_modules/mysql/_params.py

```diff
@@ -299,6 +299,20 @@ def load_arguments(self, _):    # pylint: disable=too-many-statements, too-many-
         c.argument('standby_availability_zone', arg_type=standby_availability_zone_arg_type)
         c.argument('database_name', arg_type=database_name_arg_type)
         c.argument('yes', arg_type=yes_arg_type)
+    
+    with self.argument_context('mysql flexible-server threat-protection update') as c:
```

Comment:
Can you confirm if subscription id is required or not. Maybe it is not required as az login might take care of it. Or is it a better practice to include it as an optional parameter. Please confirm.

Reply:
Yes, It is handled by az login and supplied as context.